lib/experimental/Navigation/Tabs/index.tsx

```diff
@@ -33,7 +33,7 @@ export const BaseTabs: React.FC<TabsProps> = ({
     <TabNavigation
       secondary={secondary}
       asChild
-      aria-label={secondary ? "primary-navigation" : "secondary-navigation"}
+      aria-label={secondary ? "secondary-navigation" : "primary-navigation"}
```

Comment:
Let's then update the language file and request proper translations for the labels